assets/src/scripts/_output-click.js

```diff
@@ -116,30 +116,34 @@ const outputClick = async ({ outputName, metadata, url }) => {
   if (method) {
     toggleParentVisibility("outputType", "div", "show");
     setElementText("outputType", `${method ?? ""} ${type}`);
+  } else {
+    toggleParentVisibility("outputType", "div", "hide");
```

Comment:
I find it really jarring to have the fields jump around when moving between outputs. Rather than hiding the element, could we do something like this instead?

```suggestion
    setElementText("outputType", "Custom");
```

Reply:
Yes, sure. I was trying to make the minimal change, but I agree this is better. I'm trying to avoid "custom" as a term here because I don't think it really means anything outside the ACRO context, but "unknown" would be fine I think.